packages/app-mobile/components/NoteEditor/NoteEditor.tsx

```diff
@@ -277,6 +303,11 @@ function NoteEditor(props: Props, ref: any) {
 		};
 	}, [html]);
 
+	useEffect(() => {
+		// Update the text when props.text changes (and doesn't match the current document's text).
+		webviewRef.current.injectJavaScript(`cm.setText(${JSON.stringify(props.text)});`);
+	}, [props.text]);
+
```

Comment:
We really don't want that as it will becomes very difficult to maintain. The editor manages the content and notify the parent via onChange event.

If the content needs to be changed it should be via an explicit function call.